ns/namespace.go

```diff
@@ -177,4 +178,61 @@ func FromCompositionTags(object interface{}, current string, namespace *[]string
 	return FromMap(jmap, current, namespace)
 }
 
-// TODO - Value getters (GetValueByTag, GetValueByNamespace etc)
+// GetValueByNamespace returns value stored in struct composition.
+// It requires filed tags on each struct field which may be represented as namespace component.
+// It iterates over fields recursively, checks tags until it finds leaf value.
+func GetValueByNamespace(object interface{}, ns []string) interface{} {
+	// current level of namespace
+	current := ns[0]
+	fields, err := reflections.Fields(object)
+	if err != nil {
+		fmt.Printf("Could not return fields for object{%v}\n", object)
+		return nil
+	}
+
+	for _, field := range fields {
+		tag, err := reflections.GetFieldTag(object, field, "json")
+		if err != nil {
+			fmt.Printf("Could not find tag for field{%s}\n", field)
+			return nil
+		}
+		// remove omitempty from tag
+		tag = strings.Replace(tag, ",omitempty", "", -1)
+		if tag == current {
+			val, _ := reflections.GetField(object, field)
```

Comment:
consider not suppressing but rather handling error that can be returned by GetField